### PR TITLE
Add the 2021 icra entry

### PR DESCRIPTION
**Pull request by Kelym**
- [x] Update `.bib` file
- [x] Upload publication PDF to [Google Drive](https://drive.google.com/drive/folders/1M9fOGIIQ3e1R62dtVit5rZ5iWZqxfWV9)

### CODE REVIEW

File: siddpubs-conf.bib [hunk 1]
```diff
@@ -63,6 +63,13 @@ @string{ldc
 
 % 2021 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{ke2021grasping,
+ title = {{Grasping with Chopsticks: Combating Covariate Shift in Model-free Imitation Learning for Fine Manipulation}},
+ author = {Ke, L., Wang, J., Bhattacharjee, T., Boots, B. and Srinivasa, S.S.},
```
Comment (brianhou) [inline siddpubs-conf.bib]: Use `and` to separate authors.

Comment (egordon) [inline siddpubs-conf.bib]: I think the website assumes that authors are separated by ` and `, see all other entries below.

Best to keep that consistent

**Review by brianhou — CHANGES_REQUESTED** [review]

**Review by egordon — CHANGES_REQUESTED** [review]

**Comment by egordon** [comment]
@Kelym Noticed this has been tied up for a year. Mind fixing the issues?

**Comment by Kelym** [comment]
@egordon @brianhou 
Fixed the "and". Ready to merge now.

**Review by brianhou — APPROVED** [review]

**Pull request merged**